packages/server/lib/browsers/firefox.ts

```diff
@@ -281,6 +282,8 @@ const defaultPreferences = {
   // allow getUserMedia APIs on insecure domains
   'media.devices.insecure.enabled':	true,
   'media.getusermedia.insecure.enabled': true,
+
+  'marionette.log.level': process.env.DEBUG ? 'Debug' : undefined,
```

Comment:
```suggestion
  'marionette.log.level': debug.enabled ? 'Debug' : undefined,
```

Reply:
this would be...

```suggestion
  'marionette.log.level': debug('cypress:launcher').enabled ? 'Debug' : undefined,
```

since stderr is piped in launcher:

https://github.com/cypress-io/cypress/blob/8df8bb446213265cd922f8cfa6cb18c5df9897e2/packages/launcher/lib/browsers.ts#L124-L126

but i figured process.env.DEBUG would be better than debug('cypress:launcher'), because it will still be correct even if the launcher's debug setup is changed, or if we ever move browser spawning to happen elsewhere

Reply:
ooh i can just use the `log.ts` module exported from launcher instead, i'll do that